plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/TypeEmbedding.kt

```diff
@@ -128,6 +130,15 @@ fun <R> TypeEmbedding.flatMapUniqueFields(action: (SimpleKotlinName, FieldEmbedd
     }
 }
 
+fun <R> TypeEmbedding.mapNotNullUniqueFields(action: (SimpleKotlinName, FieldEmbedding) -> R?): List<R> {
```

Comment:
Can this be obtained from `flatMapUniqueFields` by sending `r` to `listOf(r)` and `null` to `emptyList()`?